pkg/apiserver/definition.go

```diff
@@ -26,6 +26,10 @@ func (d *Definition) getAbsolutePath() string {
 	absolutePath := fmt.Sprintf("%s/%s", groupPath, d.relativePath)
 	absolutePath = strings.TrimRight(absolutePath, "/")
 
+	if absolutePath == "" {
```

Comment:
We have similar logic in gosoline/pkg/apiserver/middleware_metric.go:38
Does it make sense to unify them and look for other occurrences in this package?

Reply:
yes make sense to have a small function to do the Trim for the path if it's empty we can return `/` in a safe way it will be more clear and we don't need to duplicate!